test/test_unistd.rs

```diff
@@ -600,6 +600,62 @@ fn test_symlinkat() {
     );
 }
 
+#[test]
+fn test_getpwuid() {
+    let res = User::query( UserQuery::Uid(Uid::from_raw(1)) ).unwrap();
+    assert!(res.unwrap().uid == Uid::from_raw(1));
+}
+
+#[test]
+fn test_getgrgid() {
+    let res = Group::query( GroupQuery::Gid(Gid::from_raw(1)) ).unwrap();
```

Comment:
Are you sure that there will be a GID 1?

Reply:
Ditto.